src/pages/storage/storage-new.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <page :title="title" :breadcrumb-links="breadcrumbLinks" :errors="errors">
+    <template #header-right>
+      <button-overlay :show="processing">
+        <b-button variant="primary" v-on:click="onCreateClick">Create</b-button>
+      </button-overlay>
+    </template>
+    <div class="pr-3">
+      <div class="pt-3">
+        <label class="form-label">Hostname</label>
+        <b-form-input v-model="hostName" type="text" :state="hostnameValidation"/>
```

Comment:
Please follow the style of validation done on other pages.
https://github.com/SciGaP/emcenter-gateway/blob/master/src/pages/group/group-new-page.vue#L11

Reply:
Added the validation and disabled the create button until all fields are validated true.